test-utils/Cargo.toml

```diff
@@ -6,6 +6,31 @@ edition = "2018"
 license = "Apache-2.0"
 homepage = "https://substrate.dev"
 repository = "https://github.com/paritytech/substrate/"
+autotests = false
```

Comment:
Why?

Reply:
Because there are trybuild tests in the tests directory

Reply:
Please just implement these tests as they are done everywhere else in the codebase.